docs/nlu/components.rst

```diff
@@ -1458,204 +1527,168 @@ DIETClassifier
-            "scale_loss": True
-            # ## Regularization parameters
-            # The scale of regularization
-            "regularization_constant": 0.002
-            # The scale of how important is to minimize the maximum similarity
-            # between embeddings of different labels.
-            "negative_margin_scale": 0.8
-            # Sparsity of the weights in dense layers
-            "weight_sparsity": 0.8
-            # Dropout rate for encoder
-            "drop_rate": 0.2
-            # Dropout rate for attention
-            "drop_rate_attention": 0
-            # If 'True' apply dropout to sparse tensors
-            "use_sparse_input_dropout": True
-            # ## Evaluation parameters
-            # How often calculate validation accuracy.
-            # Small values may hurt performance, e.g. model accuracy.
-            "evaluate_every_number_of_epochs": 20
-            # How many examples to use for hold out validation set
-            # Large values may hurt performance, e.g. model accuracy.
-            "evaluate_on_number_of_examples": 0
-            # ## Model config
-            # If 'True' intent classification is trained and intent predicted.
-            "intent_classification": True
-            # If 'True' named entity recognition is trained and entities predicted.
-            "entity_recognition": True
-            # If 'True' random tokens of the input message will be masked and the model
-            # should predict those tokens.
-            "use_masked_language_model": False
-            # 'BILOU_flag' determines whether to use BILOU tagging or not.
-            # If set to 'True' labelling is more rigorous, however more
-            # examples per entity are required.
-            # Rule of thumb: you should have more than 100 examples per entity.
-            "BILOU_flag": True
+    If you want to use the ``DIETClassifier`` just for intent classification, set ``entity_recognition`` to ``False``.
+    If you want to do only entity recognition, set ``intent_classification`` to ``False``.
+    By default ``DIETClassifier`` is doing both, i.e. ``entity_recognition`` and ``intent_classification`` is set to
+    ``True``.
+
+    You can define a number of hyperparameters to adapt the model.
+    If you want to adapt your model, start by modifying the following parameters:
+
+        - ``epochs``:
+          This parameter sets the number of times the algorithm will see the training data (default: ``300``).
+          One ``epoch`` is equals to one forward pass and one backward pass of all the training examples.
+          Sometimes the model needs more epochs to properly learn.
+          Sometimes more epochs don't influence the performance.
+          The lower the number of epochs the faster the model is trained.
+        - ``hidden_layers_sizes``:
+          This parameter allows you to define the number of feed forward neural network layers and their output
```

Comment:
`feed forward neural network layers` should be changed to `feed forward layers`.
`network` here would be confusing